code/modules/SCP/SCPs/SCP-2427-3.dm

```diff
@@ -176,15 +176,15 @@ GLOBAL_LIST_EMPTY(scp2427_3s)
 	if(A in purity_list)
 		to_chat(src, SPAN_WARNING("They are pure... We will grant their wish."))
 		return
-	if(ishuman(A) && (satiety > min_satiety) && !(A in impurity_list))
+	if(ishuman(A) && (satiety > min_satiety) && !(A in impurity_list) || !isMonkey(A))
```

Comment:
This will make it so 2427-3 cannot attack any humans, since they are not monkeys. Also, wrong proc call:
Should be `A.isMonkey()`. Either way, need to fix it.
```suggestion
	if(ishuman(A) && (satiety > min_satiety) && !(A in impurity_list) && !A.isMonkey())
```
I think it won't work, since A is an atom type and isMonkey is a mob/living proc, do with that what you will.